lesson7-questions.txt

```diff
@@ -4,7 +4,7 @@ Questions for Lesson 7
 before some or all of the controller actions execute?  Where can you put that
 change so that it affects all controllers?
 
-
+Callback Method. To apply it to all controllers, I need to put it in ApplicationController.
```

Comment:
bonus follow up question: why exactly does adding it to the ApplicationController class add it to the rest of them?